custom_typings/dom.d.ts

```diff
@@ -0,0 +1,11 @@
+declare interface Window {
+  customElements: CustomElementRegistry;
```

Comment:
A [PR](https://github.com/Microsoft/TSJS-lib-generator#contribution-guidelines) would be appreciated. We will need to add the types to [addedTypes.json overrides](https://github.com/Microsoft/TSJS-lib-generator/blob/master/inputfiles/addedTypes.json).